spec.go

```diff
@@ -13,7 +13,7 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-//go:embed spec spec/integration/_dev spec/integration/data_stream/_dev spec/input/_dev
+//go:embed spec spec/integration/_dev spec/integration/data_stream/_dev spec/input/_dev docsValidationSchema
```

Comment:
Camel case is not usual in go projects, we could use something like `docs_schema`.
```suggestion
//go:embed spec spec/integration/_dev spec/integration/data_stream/_dev spec/input/_dev docs_schema
```